src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -135,6 +135,7 @@ public boolean hasDeadline(Deadline deadline) {
         return addressBook.hasDeadline(deadline);
     }
 
+    //@@author LeonardTay748
     @Override
     public void editStats(int type) {
```

Comment:
You might to break this into 3 methods for readability. The code used in other classes i.e. `editStats(1)` didn't seem very clear until I saw this